### PR TITLE
Update the Gemfile.lock for security updates.

### PR DESCRIPTION
**Pull request by dperez37**
This PR updates the gemfile.lock file to get security updates.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -29,45 +29,49 @@ GEM
...
+    kramdown (2.3.2)
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
     mini_portile2 (2.8.0)
     multipart-post (2.1.1)
     nap (1.1.0)
     no_proxy_fix (0.1.2)
-    nokogiri (1.13.3)
+    nokogiri (1.13.4)
```
Comment (byencho) [inline Gemfile.lock:67]: Just curious if you want to update everything or just nokogiri by doing `bundle update nokogiri`?

Comment (dperez37) [reply Gemfile.lock:67]: I usually try to keep everything up to date.

Comment (byencho) [reply Gemfile.lock:67]: OK that's fine. Just wanted to check.

**Review by byencho — APPROVED** [review]

**Comment by dperez37** [comment]
Thanks @byencho & @alexvanyo

**Pull request merged**